tecmint_monitor.sh

```diff
@@ -44,8 +44,8 @@ if [ ! -z "$iopt" ]; then
 	ok_msg="Congratulations! Script Installed, now run monitor Command"
 	wd=$(pwd)
 	basename "$(test -L "$0" && readlink "$0" || echo "$0")" > /tmp/scriptname
-	scriptname=$(echo -e -n "$wd"/ && cat /tmp/scriptname)
-	su -c "cp $scriptname /usr/bin/monitor" root && echo "${ok_message}" || echo "${fail_msg}"
+	scriptname=$(printf "%s" "$wd"/ && cat /tmp/scriptname)
```

Comment:
At least me I'm more used to output with echo, I learned first from the book "The UNIX Programming Environment" by Kernighan & Pike. 

Good job!!